providers/standard/tests/unit/standard/operators/test_latest_only_operator.py

```diff
@@ -115,9 +115,7 @@ def test_skipping_non_latest(self, dag_maker):
             start_date=timezone.utcnow(),
             logical_date=timezone.datetime(2016, 1, 1, 12),
             state=State.RUNNING,
-            data_interval=DataInterval(
-                timezone.datetime(2016, 1, 1, 12), timezone.datetime(2016, 1, 1, 12) + INTERVAL
-            ),
+            data_interval=DataInterval(timezone.datetime(2016, 1, 1, 12), timezone.datetime(2016, 1, 1, 12)),
```

Comment:
This is the crux of the test change -- interval start = interval end, which is what you get by default when you create the scheduler creates DagRuns from a DAG with a `timedelta` schedule.